Calling/ClientApp/src/components/ChatThread.tsx

```diff
@@ -302,6 +307,22 @@ export default (props: ChatThreadProps): JSX.Element => {
     setMessagesWithAttachedRef(newMessagesWithAttached);
   };
 
+  const isUploadedFileMessage = (messageContent: string): Boolean => {
+    let messageContentJson = undefined;
+    try {
+      messageContentJson = JSON.parse(messageContent);
+    } catch (error) {
+      // Not a file upload event.
+      return false;
+    }
+
+    if (messageContentJson.event && messageContentJson.event !== "FileUpload") {
```

Comment:
Should the condition be `!messageContentJson.event || messageContentJson.event !== "FileUpload"`? Otherwise, a valid JSON without an `event` property would return `true`.

Reply:
fixed by returning the opposite of this condition